### ISSUE TITLE
[Regression] Runtime error introduced when build using SDK 5.0.301

### ISSUE THREAD

**Issue by synercoder** [dotnet/sdk#18229]
**TL;DR:** Bug introduced when building using latest SDK (5.0.301), 5.0.300 was fine. Links to bug repo at the bottom.

---

I work at a printing company, and to create order tickets etc, I create a thumbnail of the PDF to be printed and make an orderticket.

Yesterday I made a small improvement in a totally unrelated part of the code, and suddenly I got crashes in my app with `AccessViolationException`. I narrowed the error down, and found out that the only change was the SDK used. I try to keep my code in the company on the latest version, so my (Azure Devops) pipeline used SDK version 5.0.x.

So my latest build (that kept crashing during runtime) was made with version 5.0.301. The previous version that did not crash was build using 5.0.300.

I recreated the bug in the smallest code I could, it involves calling PDFIUM (chromium pdf engine) using a library I originally stripped down to just the rendering part.

Recreated bug code: https://github.com/synercoder/AccessViolationBug
GitHub Action showing test error on latest SDK version: https://github.com/synercoder/AccessViolationBug/actions/runs/928660501

![Image shows build result](https://user-images.githubusercontent.com/18616295/121706353-d7a9b400-cad5-11eb-9b9e-708be1914448.png)

**Comment by synercoder** [comment]
Smoal update: workaround for me is to force 5.0.300 in the pipeline and/or using a global.json to force 5.0.300. So not all hope was lost for me.

**Comment by synercoder** [comment]
Update on the update, you can't install 5.0.300 when 5.0.301 is already installed. global.json is not an option. Install 5.0.300 in the pipeline seems to do the trick to get working builds.

**Comment by synercoder** [comment]
Fixed in later release
